src/libraries/System.Text.Json/tests/Resources/Strings.resx

```diff
@@ -20278,4 +20278,7 @@ tiline\"another\" String\\"],"str":"\"\""}</value>
   <data name="LoremIpsum40WordsBase64" xml:space="preserve">
     <value>TG9yZW0gaXBzdW0gZG9sb3Igc2l0IGFtZXQsIGNvbnNlY3RldHVyIGFkaXBpc2NpbmcgZWxpdC4gQ3VyYWJpdHVyIHZpdmVycmEgbmVxdWUgYXQgZXJhdCBsYW9yZWV0LCBhdCBzYWdpdHRpcyBhcmN1IGVmZmljaXR1ci4gVXQgcHVsdmluYXIgZXJvcyBuZWMgb2RpbyBjdXJzdXMgZWxlaWZlbmQuIE1hZWNlbmFzIHZpdmVycmEgZWxlbWVudHVtIHBvcnR0aXRvci4gTnVsbGFtIG1pIHZlbGl0LCBtYWxlc3VhZGEgY29tbW9kbyB0cmlzdGlxdWUgZXUsIG1vbGxpcyBhIHZlbGl0LiBNb3JiaS4=</value>
   </data>
-</root>
+  <data name="BufferMaximumSizeExceeded" xml:space="preserve">
```

Comment:
Just to confirm, was adding this to the tests project necessary? It probably is given we include the `ArrayBufferWriter.cs` file, but curios if you saw a build failure without it and then added it or added it preemptively.

Reply:
Yes, it was failing: https://github.com/dotnet/runtime/pull/32587/commits/0f4bc7984ff461c22f714cd1068ef87c80cab509
https://github.com/dotnet/runtime/runs/530510806